app/actions/views/channel.js

```diff
@@ -537,9 +581,11 @@ export function increasePostVisibility(channelId, focusedPostId) {
 
         let result;
         if (focusedPostId) {
-            result = await getPostsBefore(channelId, focusedPostId, page, pageSize)(dispatch, getState);
+            result = await dispatch(getPostsBefore(channelId, focusedPostId, page, pageSize));
+        } else if (getPostsRequest.status === RequestStatus.STARTED) {
```

Comment:
Can we check this earlier in the function?

I don't like having an early return mixed in with some other cases that are just loading data, and we skip some other stuff by moving this earlier